browse/controllers/stats_page/__init__.py

```diff
@@ -121,6 +121,8 @@ def get_monthly_submissions_page() -> Response:
     try:
         num_submissions, historical_delta = \
             get_monthly_submission_count()
+        num_this_month = get_document_count_by_yymm(current_dt.date)
```

Comment:
Are these changes intended to be part of this PR or are the part of some other work?

Reply:
Not originally. This a "miscellaneous fix" for a bug that I just happened to notice while I was working on this other stuff.